README.md

```diff
@@ -18,22 +18,27 @@
 
 <!-- start elevator-pitch -->
 
-Fine-tuning is an effective way to improve the performance on neural search tasks. However, it is non-trivial for many deep learning engineers.
+Fine-tuning is an effective way to improve performance on neural search tasks. However, it can be very difficult for
+many deep learning engineers to do.
 
-Finetuner makes fine-tuning easier, faster and performant by streamlining the workflow and handling all complexity and infrastructure on the cloud.
-With Finetuner, one can easily uplift pre-trained models to be more performant and production ready.
+Jina AI's Finetuner makes fine-tuning easier and faster by streamlining the workflow and handling all complexity and 
+infrastructure in the cloud. With Finetuner, one can easily enhance the performance of pre-trained models, making them 
+production-ready without buying expensive hardware.
 
-📈 **Performance promise**: uplift pretrained model and deliver SOTA performance on domain-specific neural search applications.
+📈 **Performance promise**: enhance the performance of pre-trained models and deliver state-of-the-art performance on 
```

Comment:
we could link the blog no?

Reply:
Mixed feelings about that. I'm not sure we should commit a link to the blog here. I worry about unstable links. Especially if we're going to recast the README in the future to match the new direction for marketing Finetuner.